test/libsolidity/smtCheckerTests/functions/functions_trivial_condition_for.sol

```diff
@@ -5,4 +5,4 @@ contract C
 	function f(bool x) public pure { require(x); for (;x;) {} }
 }
 // ----
-// Warning: (98-99): For loop condition is always true.
+// Warning: (98-99): Condition is always true.
```

Comment:
The message is less specific now due to these cases sharing the same target `ConstantCondition`